CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributor Guideline
+
+
+## Commit
+
+> One golden rule: follow [the commit format][commit-format]
+
+We use [standard-version][standard-version] to automatically generate [CHANGELOG.md][CHANGELOG.md] from the `git commit` history. Please **do** follow [the commit format][commit-format] before commiting.
+
+
+## PR
+
+Simple rules:
+
+* Do rebase before merging.
```

Comment:
@tomchentw Sorry if I ask more questions, but I am not familiar with rebasing process to close a PR. My process usually uses `git merge` through the following three steps: 

1. merge alpha into branch
2. resolve conflicts
3. marge branch into master (github merge button. PR here gets closed).

Your flow instead doesn't need step 1 and 2, and instead of merging in step 3, we do a rebase. So we don't use the merge button from github. Am I correct?